tests/Dumplie/Test/Integration/Infrastructure/Doctrine/Metadata/DoctrineStorageTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Test\Integration\Infrastructure\Doctrine\Metadata;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DriverManager;
+use Dumplie\Application\Metadata\Schema;
+use Dumplie\Application\Metadata\Schema\Field\TextField;
+use Dumplie\Application\Metadata\Schema\TypeSchema;
+use Dumplie\Infrastructure\Doctrine\Metadata\DoctrineStorage;
+use Dumplie\Infrastructure\Doctrine\Metadata\Field\TextMapping;
+
+class DoctrineStorageTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * @var Schema
+     */
+    private $schema;
+
+    /**
+     * @var DoctrineStorage
+     */
+    private $storage;
+
+    public function setUp()
+    {
+        $this->connection = DriverManager::getConnection(['url' => 'sqlite:///:memory:']);
+
+        if ($this->connection->getSchemaManager()->tablesExist(DoctrineStorage::TABLE_PREFIX . '_sample_meta')) {
+            $this->connection->getSchemaManager()->dropTable(DoctrineStorage::TABLE_PREFIX . '_sample_meta');
+        }
+
+        $type = new TypeSchema('meta', ['id' => new TextField(), 'field' => new TextField()]);
+
+        $this->schema = new Schema('sample');
+        $this->schema->add($type);
+
+        $this->storage = new DoctrineStorage($this->connection, [new TextMapping()]);
+    }
+
+    public function testCreate()
+    {
+        $this->storage->create($this->schema);
```

Comment:
what will happend in case where schema is already created but you run Storage::create anyway?